SEC/Debt/TradedShortTermDebt.rdf

```diff
@@ -97,12 +99,19 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">bankers&apos; acceptance</rdfs:label>
-		<skos:definition xml:lang="en">a short-term debt instrument issued by a company that is guaranteed by a commercial bank</skos:definition>
+		<skos:definition xml:lang="en">short-term debt instrument issued by a company that is guaranteed by a commercial bank</skos:definition>
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Considered negotiable instruments with features of a time draft, bankers&apos; acceptances are created by the drawer and provide the bearer with the right to the amount noted on the face of the acceptance on the specified date. Unlike traditional checks, bankers&apos; acceptances function based on the creditworthiness of the banking institution instead of the individual or business acting as the drawer. Additionally, the drawer must provide the funds necessary to support the bankers&apos; acceptance, eliminating the risk associated with insufficient funds on the part of the drawer.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-sec-dbt-tstd;BillOfExchange">
-		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-dbti;FixedIncomeSecurity"/>
+		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-dbti;TradableDebtInstrument"/>
```

Comment:
we still need to address the redundancy that we have TradableX (class) and isNegotiable (boolean property) when they mean the same thing. That would make sense as a separate issue but we'd need to reflect the choice here.